src/azul/chalice.py

```diff
@@ -80,6 +81,14 @@ class ServiceUnavailableError(ChaliceViewError):
     STATUS_CODE = 503
 
 
+@attr.s(auto_attribs=True, frozen=True, kw_only=True)
+class MetricThreshold:
+    lambda_name: str
+    handler_name: Optional[str]
```

Comment:
Nitpick. Saves one line elsewhere by making the parameter optional.
```suggestion
    handler_name: Optional[str] = attr.ib(default=None)
```